state/event_table_test.go

```diff
@@ -924,3 +925,168 @@ func TestEventTableSelectUnknownEventIDs(t *testing.T) {
 		}
 	}
 }
+
+func TestEventTableRedact(t *testing.T) {
+	db, close := connectToDB(t)
+	defer close()
+	table := NewEventTable(db)
+	txn, err := db.Beginx()
+	if err != nil {
+		t.Fatalf("failed to start txn: %s", err)
+	}
+	defer txn.Rollback()
+
+	testCases := []struct {
+		original    map[string]interface{}
+		roomVer     string
+		wantContent map[string]interface{}
+	}{
+		// sanity check
+		{
+			original: map[string]interface{}{
+				"type":   "m.room.message",
+				"sender": "@nasty-mc-nastyface:localhost",
+				"content": map[string]interface{}{
+					"msgtype": "m.text",
+					"body":    "Something not very nice",
+				},
+			},
+			wantContent: map[string]interface{}{},
+			roomVer:     "10",
```

Comment:
suggest a test case that checks room version 11, since there is explicit logic for dealing with that room ver

Reply:
Gomatrixserverlib does not support v11.